### ISSUE TITLE
Usage confusing

### ISSUE THREAD

**Issue by Hsanka6** [Ramotion/expanding-collection#50]
Can someone help me with integrating this with my app?

**Comment by heuism** [comment]
Yeah me as well. I followed the usage but can't find the way to make it work.

Please help @RamotionRussell

**Comment by waseefakhtar** [comment]
Same here. It’s been hours now and I can’t get my head around it. This is a great project. However, the README file needs to be clearly written on how to install, setup, and use ExpandingCollection.

**Comment by erhiesfeka** [comment]
The documentation could be improved for sure. I found it helpful to download and follow the example in the source code. Check to make sure that all the IBOutlets are hooked up correctly and use the example in the source for reference. Also make sure that the constraints are for the front and back views are as per the documentation. That being said @RamotionRussell , could the README be updated please :)

**Comment by atoaster** [comment]
For future reference, someone was good enough to make a Swift 3 Fork with a demo application here:

https://github.com/arch1t3ct/expanding-collection/tree/swift3

**Comment by guru-teja** [comment]
Is there any other tutorial or example that can help in better understanding of using the project?
